cloud/amazon/ec2_metric_alarm.py

```diff
@@ -184,8 +184,11 @@ def create_metric_alarm(connection, module):
         comparisons = {'<=' : 'LessThanOrEqualToThreshold', '<' : 'LessThanThreshold', '>=' : 'GreaterThanOrEqualToThreshold', '>' : 'GreaterThanThreshold'}
         alarm.comparison = comparisons[comparison]
 
-        dim1 = module.params.get('dimensions', {})
+        dim1 = module.params.get('dimensions')
```

Comment:
I don't understand this change at all. This code appears to be equivalent to the previous code, but less idiomatic.